### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by rdbende** [tukaan/tukaan#118] [Announcements]
Lots of breaking changes.

## What's Changed
* change: use properties & descriptors by @demberto in https://github.com/tukaan/tukaan/pull/104
* cleanup by @rdbende in https://github.com/tukaan/tukaan/pull/105
* Bit of refactor on window stuff by @rdbende in https://github.com/tukaan/tukaan/pull/106
* Add theming by @rdbende in https://github.com/tukaan/tukaan/pull/108
* Fix a lotta bugs, add some tests by @rdbende in https://github.com/tukaan/tukaan/pull/110
* Add filedialogs by @rdbende in https://github.com/tukaan/tukaan/pull/89
* Check non-pip dependencies in `setup.py` by @rdbende in https://github.com/tukaan/tukaan/pull/117

## New Contributors
* @demberto made their first contribution in https://github.com/tukaan/tukaan/pull/104

**Full Changelog**: https://github.com/tukaan/tukaan/compare/v0.1.3...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tukaan/tukaan/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by sumeshir26** [comment]
So how many more breaking changes are expected?

**Comment by sumeshir26** [comment]
So how many more breaking changes are expected?

**Reply by rdbende** [reply]
Good question. I don't know.

**Reply by rdbende** [reply]
Since I use semantic version numbering, you can expect bigger API changes, until v1.0.0 is released.

**Reply by Moosems** [reply]
Do you think we'll get a stand in place/temporary tk Text widget?

**Reply by rdbende** [reply]
Yes, I'll make it right after I finished the canvas widget(s).

**Reply by Moosems** [reply]
How's that going?

**Comment by rdbende** [comment]
v0.2.0 didn't work. I now released v0.2.1 which fixes a fatal error on initialization.

**Reply by Moosems** [reply]
Nice 👍
